latex_access/tests/test_settings.py

```diff
@@ -0,0 +1,40 @@
+import unittest
+import latex_access
+from latex_access import settings as settings_module
+
+try:
+    settings_module.settings['brailletable'] = 'nemeth'
+    settings_module.brailleTableToUse()
+    settings_module.settings['brailletable'] = 'ueb'
+    settings_module.brailleTableToUse()
+    incompatible = False
+except AttributeError:
+    incompatible = True
+
+class TestSettings(unittest.TestCase):
+    def test_load_settings(self):
+        """Tests that settings are read from a file."""
+        settings_module.settings = {}
+        settings_module.loadSettings('latex_access/tests/test_settings.txt')
+        self.assertEqual(settings_module.settings, {'brailledollars': 'true', 'speakdollars': 'true', 'capitalisation':'6dot'})
+
+    def test_booleanise_setting(self):
+        """Tests that setting's value is converted to boolean."""
+        settings_module.settings['brailledollars'] = 'True'
+        settings_module.settings['foo'] = 'bar'
+        self.assertTrue(settings_module.booleaniseSetting('brailledollars'))
+        self.assertFalse(settings_module.booleaniseSetting('foo'))
+
+    def test_get_setting(self):
+        """Tests that value is returned for given setting."""
+        settings_module.settings['speakdollars'] = True
+        self.assertTrue(settings_module.getSetting('speakdollars'))
+        self.assertFalse(settings_module.getSetting('foo'))
```

Comment:
Since you're manipulating the default settings and do not restore them after each test the `else` block is not tested (`False` is returned because `booleaniseSetting` returns `False` for `"bar"` and not because settings is missing in the dict. If you would restore the default values in `tearDown` this problem would be gone.